src/services/user.service.js

```diff
@@ -37,3 +38,35 @@ export const createUser = async (req, res) => {
 	}
 };
 
+
+export const loginUser= async (req, res) => {
```

Comment:
Please move this codes into Authentication controller as registration is in authentication controller